packages/boilerplate/composables/src/composables/getters/productGetters.ts

```diff
@@ -28,14 +28,26 @@ export const getProductPrice = (product: ProductVariant): AgnosticPrice => {
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 export const getProductGallery = (product: ProductVariant): AgnosticMediaGalleryItem[] => [
   {
-    small: 'https://s3-eu-west-1.amazonaws.com/commercetools-maximilian/products/081223_1_large.jpg',
-    normal: 'https://s3-eu-west-1.amazonaws.com/commercetools-maximilian/products/081223_1_large.jpg',
-    big: 'https://s3-eu-west-1.amazonaws.com/commercetools-maximilian/products/081223_1_large.jpg'
+    mobile: {
```

Comment:
I think we have talked about adding another computed property on product page, not about changing the getters data formats, this part has to be reverted in all integrations